tdrs-backend/tdpservice/users/migrations/0012_add_initial_super_user.py

```diff
@@ -0,0 +1,25 @@
+import os
+from django.db import migrations
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('users', '0011_auto_20210108_1741'),
+    ]
+
+    def generate_superuser(apps, schema_editor):
+        from ..models import User
```

Comment:
```suggestion
        User = apps.get_model('users', 'User')
```

When importing models in migrations code it is important to use the historical versions of the model exposed by the `apps` argument, to prevent issues in the future running this migration if the application code changes in some way. (For example, if we add or remove fields on the custom User model)

https://docs.djangoproject.com/en/3.1/topics/migrations/#historical-models

Reply:
Good catch!